docs/user-guide/glossary.md

```diff
@@ -181,6 +181,12 @@ quantified expression
   referred to as *quantified expressions*.  See {ref}`logic-exprs` for
   details about quantifiers in CVL.
 
+receiveOrFallback
+  A special function we introduce in every contract to model the behavior of solidity
+  for calls with no data or that do not resolve to any contract function.
```

Comment:
I think we should we add a link to the official EVM docs here?

Reply:
Something like this? https://docs.soliditylang.org/en/latest/contracts.html#fallback-function

Reply:
Exactly 👍